app/src/main/res/layout/fragment_about.xml

```diff
@@ -158,6 +158,17 @@
                     android:layout_marginRight="@dimen/spacing_negative"
                     android:layout_marginStart="@dimen/spacing_negative" />
 
+                <TextView
+                    android:id="@+id/txt_github_repository"
+                    style="@style/AboutRow"
+                    android:text="@string/about_github_repository" />
```

Comment:
Could you change `android:text` to this?
`app:textRtlConsidered="@string/about_github_repository"`

Because in the case of Arabic locale, Text direction should be Right To Left. But when first character of the text is NOT Arabic (For example `GitHub مستودع`), RTL is ignored! :cry: So that custome attribute `app:textRtlConsidered` force RTL to TextView.

I created this attributes using DataBinding. To know more, See [DataBindingUtil.java](https://github.com/konifar/droidkaigi2016/blob/master/app/src/main/java/io/github/droidkaigi/confsched/util/DataBindingAttributeUtil.java#L103-L106).